docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,110 @@
+
+  ✅ Save <br>
+  ✅ Next <br>
+  ✅ Save draft <br>
+  ✅ Next step <br>
+  ⛔️ Save a draft
+
+- Avoid labels that are too generic, such as `Yes` or `No`. They are not intuitive and require the user to read the supporting text.  
+- Use sentence case as it reads more natural and friendly to the users, which makes them more willing to press the button. Title case doesn't read as easily as sentence case and looks more unnatural.
+
+  ✅ Next section <br>
+  ⛔️ Next Section
+
+- Be careful when using words of similar meaning, such as `Delete` and `Remove`.
+
+
+## Tooltips
+
+Tooltips are messages that provide additional information about a certain UI element. Still, the text within a tooltip should contain information that brings value to the user, so avoid adding content that may be irrelevant. Tooltips shouldn't also contain information that is necessary to complete a given task. Such information should be visible at first glimpse. Otherwise, users will have to remember the content of a tooltip and refer back to it after the tooltip is deactivated. Here are some general guidelines on creating tooltip texts:
+
+- Keep your tooltips 1-2 sentences long. If you need to explain a concept in more details, create a separate document and link to it instead of providing the whole explanation in a tooltip.
+- Use sentence case inside a tooltip and follow punctuation rules.
+- Use tooltips to provide examples, for example:
+
+  ✅ The example of a domain name is `yourdomain.example.com`.
+
+- Omit punctuation in case of one-line label tooltips. For more information, see [Labels](#labels).
+
+
+## Messages
+
+There are many different types of messages appearing in every UI. The most common ones are pop-ups and regular messages that  users can find on your page. To avoid information pollution, make sure that all your messages are informative and useful to the users. Moreover:
+
+- Use punctuation in case of messages that are full sentences. In case of clauses (fragments of a sentence), omit punctuation.
+- Use title case for the title of your pop-up message. Keep it as simple as possible and omit punctuation. For more information, see [Labels](#labels).
+
+### Error messages
+
+Error message is a special type of message you must be really careful about. It informs about unsuccessful outcome of an action or conveys any other negative information, so you must choose your words wisely not to upset the user. Here are some tips that will help you design your error message:  
+
+- Describe what happened, why it happened, and what the user can do to fix the occurred issue. Avoid abstract messages that only inform that something went wrong. This could make the user feel frustrated not only about the fact that the error has occurred, but also for the fact they cannot do anything about it. However, do not describe the whole troubleshooting in an error message. Link to a separate troubleshooting guide instead.
```

Comment:
```suggestion
- Describe what happened, why it happened, and what the user can do to fix the issue. Avoid abstract messages that only inform that something went wrong. This could make the user feel frustrated not only about the fact that the error has occurred, but also for the fact they cannot do anything about it. However, do not describe the whole troubleshooting in an error message. Link to a separate troubleshooting guide instead.
```